### DISCUSSION TITLE
Make a floating button for bottom navigation (border opacity problem)

### DISCUSSION THREAD

**Discussion by try333** [tailwindlabs/tailwindcss#12842] [Help]
does anyone know how to make this tranparent see through border things??
![Screenshot 2024-01-26 230503](https://github.com/tailwindlabs/tailwindcss/assets/29312095/9314e4df-62ae-4dc4-9883-ab0bdfb90e5b)

**Comment by makibat** [comment]
You could just set the border to be the [same color](https://play.tailwindcss.com/eKuCZ5vl9a) as your background, if that works for your case.
